packages/office-ui-fabric-react/src/components/Keytip/examples/Keytip.Disabled.Example.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { IKeytipProps, Keytip } from 'office-ui-fabric-react/lib/Keytip';
+import { convertSequencesToKeytipID } from 'office-ui-fabric-react/lib/Utilities';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { IKeytipExampleState, onKeytipButtonClick } from './Keytip.Basic.Example';
+
+export interface IKeytipMap {
+  [componentKeytipId: string]: IKeytipProps;
+}
+
+export class KeytipDisabledExample extends React.Component<{}, IKeytipExampleState> {
+  private keytipMap: IKeytipMap = {};
+
+  constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      keytipVisible: false
+    };
+
+    // Setup keytips
+    this.keytipMap.Keytip1 = {
+      content: 'B',
+      keySequences: ['b'],
+      disabled: true
+    } as IKeytipProps;
```

Comment:
I usually find it cleaner to do this kind of initialization in the property definition, same for the state initialization. That would also allow you to get rid of the need for a constructor.